src/app/staticanalysis/modules/staticAnalysisCommon.js

```diff
@@ -0,0 +1,156 @@
+'use strict'
+
+var utils = require('../../utils')
+
+var nodeTypes = {
+  IDENTIFIER: 'Identifier',
+  MEMBERACCESS: 'MemberAccess',
+  FUNCTIONCALL: 'FunctionCall'
+}
+
+var basicTypes = {
+  UINT: 'uint256',
+  BOOL: 'bool',
+  ADDRESS: 'address',
+  BYTES32: 'bytes32'
+}
+
+var basicRegex = {
+  CONTRACTTYPE: '^contract ',
+  FUNCTIONTYPE: '^function \\('
+}
+
+var basicFunctionTypes = {
+  SEND: buildFunctionSignature([basicTypes.UINT], [basicTypes.BOOL], false),
+  CALL: buildFunctionSignature([], [basicTypes.BOOL], true),
+  DELEGATECALL: buildFunctionSignature([], [basicTypes.BOOL], false)
+}
+
+var lowLevelCallTypes = {
+  CALL: { ident: 'call', type: basicFunctionTypes.CALL },
+  CALLCODE: { ident: 'callcode', type: basicFunctionTypes.CALL },
+  DELEGATECALL: { ident: 'delegatecall', type: basicFunctionTypes.DELEGATECALL },
+  SEND: { ident: 'send', type: basicFunctionTypes.SEND }
```

Comment:
If you only want to check those which do not throw, then `transfer` is excluded.